app/assets/javascripts/discourse/app/lib/sidebar/common/community-section/everything-section-link.js

```diff
@@ -63,7 +63,9 @@ export default class EverythingSectionLink extends BaseSectionLink {
   }
 
   get route() {
-    return "discovery.latest";
+    return this.currentUser?.sidebarTopicDestination === "unread"
```

Comment:
I am confused ... the destination can be new / unread or latest ? not just new and latest?

Reply:
I did it because of `enum sidebar_topic_destination: { latest: 0, unread: 1 }, _scopes: false`

I cannot use `new` as enum is creating dynamic methods and Rails was warning me that cannot use `new`

Reply:
I follow but the "list destination" going to be `unreadOrNew`. and the route it his should be `new` , `unread` or `latest` depending on counts. 

I would actually call things "Unread/New" cause we prioritize Unread at Discourse so we should probably call it out in the right order.